src/js/auth/containers/AuthApp.jsx

```diff
@@ -84,7 +84,7 @@ class AuthApp extends React.Component {
         <div>
           <h3>We are sorry that we could not successfully log you in.</h3>
           <h3>Please call the Vets.gov Help Desk at 1-855-574-7286. We're open Monday‒Friday, 8:00 a.m.‒8:00 p.m. (ET).</h3>
-          <button onClick={window.close()}>Close</button>
+          <button onClick={window.close}>Close</button>
```

Comment:
This is fine, but it makes me a little antsy because you would normally want to bind this function so it has the right `this` value when executed (e.g. `window.close.bind(window)`). It doesn't cause any problems here because `window` is the global context, I think.